.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -103,6 +103,16 @@ body:
         - 'TypeScript (plugin, built-in types)'
     validations:
       required: true
+  - type: dropdown
+    attributes:
+      label: Which stage(s) are affected? (Select all that apply)
+      multiple: true
+      options:
+        - 'next dev'
+        - 'next build'
+        - 'next start'
```

Comment:
Should this also have an option for when deployed in production, maybe `Vercel` and then also `other` as well?

Reply:
@ijjk What do you think about this:

```
options:
  - 'next dev (local)'
  - 'next build (local)'
  - 'next start (local)'
  - 'Vercel (Deployed)'
  - 'Other (Deployed)'
```

Reply:
Sounds good to me 👍